src/TextLineSpacingCommand.ts

```diff
@@ -124,22 +125,23 @@ class TextLineSpacingCommand extends UICommand {
   };
 
   isEnabled = (state: EditorState): boolean => {
-    return this.isActive(state) || this.execute(state, undefined, undefined);
+    return this.isActive(state) || this.execute(state);
   };
 
   execute = (
     state: EditorState,
     dispatch?: (tr: Transform) => void,
     _view?: EditorView
   ): boolean => {
+    dispatch = dispatch || noop;
     const { schema, selection } = state;
     const tr = setTextLineSpacing(
       state.tr.setSelection(selection),
       schema,
       this._lineSpacing
     );
     if (tr.docChanged) {
-      dispatch && dispatch(tr as Transaction);
+      dispatch(tr as Transaction);
```

Comment:
Typescript has a null safe noop. Replace all instances of noop with dispatch?.(tr)